brave/brave-ios
# Fix #3079: Add 1.22 translations.

## Summary of Changes


This pull request fixes #3079 

## Submitter Checklist:


## Test Plan:

1. Set language to non-english one
2. Open new Rewards panel, verify text is translated
3. Scroll down to see the Brave Today intro card, verify it's translated

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
